### ISSUE TITLE
Program displays empty output boxes

### ISSUE THREAD

**Issue by MathiusEire** [IoBuilders/truffle-contract-size#12]
as stated the output of the program after it runs is just 2 empty output boxes with their corresponding contract and size labels

**Comment by daniel-iobuilders** [comment]
Hi @MathiusEire 
this happens when the plugin does not find any contracts. Is the project where this happens open source? It would help a lot to see the configuration of the project to narrow down the bug.

**Comment by MathiusEire** [comment]
The project I’m working off of is the github “optimism-box”  , first result
in google most likely

**Comment by daniel-iobuilders** [comment]
I executed the following commands:
```
mkdir optimism-box
cd optimism-box
truffle unbox optimism-box
npm install truffle-contract-size
# add plugin to truffle-config.js
npx truffle compile
npx truffle run contract-size
```

Then I got the following result:
```
│ Contract                                                             │ Size     │
│ SimpleStorage                                                   │ 0.19 KiB │
```

So it seems to work. Have you maybe not compiled the contracts before executing `truffle run contract-size`? The contracts need to  be compiled first in order to calculate the contract size. If not an empty table is shown.

**Comment by MathiusEire** [comment]
If I was to take a guess I’d imagine it’s because the program only checks
the contracts compiled with the evm rather then the ovm I’m currently
exploring

**Comment by daniel-iobuilders** [comment]
I did not realize that there is another truffle configuration file in the project for the ovm. You have to specify it when running any truffle command if not it will always use the default configuration, which points to the EVM contracts. With the following commands it works:

```
npm install truffle-contract-size
# add plugin configuration to truffle-config.ovm.js
npm run compile:ovm
npx truffle run contract-size --config truffle-config.ovm.js
```

Then I get:
```
Contract                                                         │ Size      │
SimpleStorage                                               │ 0.45 KiB |
```

**Comment by MathiusEire** [comment]
Yeah that makes more sense , that I really a contract won’t have enough gas
on the ovm/evm due to the 24kb limit right ?
